src/main/resources/db/migration/V1_1__add_created_by_user_id.sql

```diff
@@ -0,0 +1,2 @@
+alter table referral
+    add column created_by_userid text;
```

Comment:
For the data users we will have to establish where these users come from. Is it possible to capture the auth source here, either as a separate column, table or JSONB payload?

Reply:
Or by encoding it into the name and switching to integer?

`created_by_delius_user_id integer`?

Reply:
do we need to store that info, or just output in our events?

Reply:
I think so... In this use case:
- Someone looking at our data outside our application for reporting purposes
- They need to establish who this identity is
- We don't have a user table -- so this is an external reference -- but there's no context where it points